### PR TITLE
Translate Meta Instance methods part within migration  file to Spanish

### PR DESCRIPTION
**Pull request by FLarra**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
 </div>
 {% endraw %}
 
-## Meta Instance Methods
+## Métodos de metainstancia
```
Comment (Tintef) [inline src/v2/guide/migration.md]: What do you think about changing `metainstancia` to `meta instancia` or `meta-instancia`?

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
...
-No real use. If you do happen to rely on this feature somehow and aren't sure how to work around it, post on [the forum](https://forum.vuejs.org/) for ideas.
+Sin uso real. Si de alguna manera confía en esta característica y no está seguro de cómo solucionarla, publique en el [foro](https://forum.vuejs.org/) para obtener ideas.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>vm.$eval</code>. If you miss any, you'll see <strong>console errors</strong>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> su código para encontrar ejemplos de <code>vm.$eval</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Missing `en`
```suggestion
    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de <code>vm.$eval</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```

File: src/v2/guide/migration.md [hunk 3]
```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
...
-No real use. If you do happen to rely on this feature somehow and aren't sure how to work around it, post on [the forum](https://forum.vuejs.org/) for ideas.
+Sin uso real. Si de alguna manera confía en esta característica y no está seguro de cómo solucionarla, publique en el [foro](https://forum.vuejs.org/) para obtener ideas.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>vm.$interpolate</code>. If you miss any, you'll see <strong>console errors</strong>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> su código para encontrar ejemplos de <code>vm.$interpolate</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Same here 😅

```suggestion
    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de <code>vm.$interpolate</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```

File: src/v2/guide/migration.md [hunk 4]
```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
...
+  </p>
 </div>
 {% endraw %}
 
-### `vm.$log` <sup>removed</sup>
+### `vm.$log` <sup>removida</sup>
 
-Use the [Vue Devtools](https://github.com/vuejs/vue-devtools) for the optimal debugging experience.
+Use las [Vue Devtools](https://github.com/vuejs/vue-devtools) para obeten una experiencia de depuración óptima.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Typo: `obeten`

```suggestion
Use las [Vue Devtools](https://github.com/vuejs/vue-devtools) para obtener una experiencia de depuración óptima.
```

File: src/v2/guide/migration.md [hunk 5]
```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
...
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>vm.$log</code>. If you miss any, you'll see <strong>console errors</strong>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> su código para encontrar ejemplos de <code>vm.$log</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Same here 😅 

```suggestion
    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de <code>vm.$log</code>. En caso de omitir alguno, usted verá <strong>errores en la consola</strong>.
```

Comment (FLarra) [reply src/v2/guide/migration.md]: Thanks for pointing that out, this happens when you use multi-cursors to rewrite many lines at the same time

File: src/v2/guide/migration.md (hunk 1 above)
Comment (FLarra) [reply src/v2/guide/migration.md]: I like your suggestion, I'll do the change asap. Thanks for the review

**Comment by FLarra** [comment]
@Tintef addressed your last comment, thanks for the review 😃

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**